app/src/hydrate.ts

```diff
@@ -71,7 +71,7 @@ export async function hydrate(): Promise<void> {
 
 		let lang = 'en-US';
 		if (serverStore.info?.project?.default_language) lang = serverStore.info.project.default_language;
-		if (userStore.currentUser?.language) lang = userStore.currentUser?.language;
+		if (userStore.currentUser && 'language' in userStore.currentUser) lang = userStore.currentUser.language;
```

Comment:
@rijkvanzanten Looks like this broke the App for me as my users language is equal to `null` and the updated condition doesn't check if the value is truthy any more.

Reply:
Addressing this in #18493 👍